packages/fxa-settings/src/components/PageChangePassword/index.tsx

```diff
@@ -124,142 +131,167 @@ export const PageChangePassword = ({}: RouteComponentProps) => {
   };
 
   return (
-    <FlowContainer title="Change Password">
-      {alertBarRevealed && alertText && (
-        <AlertBar onDismiss={hideAlertBar} type="error">
-          <p data-testid="sign-out-error">Error text TBD. {alertText}</p>
-        </AlertBar>
-      )}
-      <form onSubmit={handleSubmit(onFormSubmit)}>
-        <h1>Stay safe — don't reuse passwords. Your password:</h1>
+    <Localized id="pw-change-header" attrs={{ title: true }}>
```

Comment:
This file is mostly just indention changes due to adding the `<Localization..>` tags.